props/saber_BC_buttons.h

```diff
@@ -364,8 +413,20 @@ Turn OFF without postoff - Turn OFF while pointing up.
 #define THRUST_GESTURE
 #endif
 
+#ifndef MAIN_BLADE
+#define MAIN_BLADE 1
+#endif
+
+#ifndef SECOND_BLADE
+#define SECOND_BLADE 2
+#endif
+
+#if defined(MAIN_BLADE) && defined(SECOND_BLADE)
+static_assert(MAIN_BLADE != SECOND_BLADE, "MAIN_BLADE and SECOND_BLADE cannot be the same");
```

Comment:
While this works, I think it might be cleaner to just do:

#if MAIN_BLADE == SECOND_BLADE
#error ....
#endif